Lib/idlelib/configdialog.py

```diff
@@ -2036,11 +2036,6 @@ def load_tab_cfg(self):
             'main', 'Indent', 'num-spaces', default=4, type='int')
         self.space_num.set(space_num)
 
-    def var_changed_space_num(self, *params):
-        "Store change to indentation size."
-        value = self.space_num.get()
-        changes.add_option('main', 'Indent', 'num-spaces', value)
```

Comment:
This should not have been deleted here or in the original.  It was an error in the test that it could still pass.